### PR TITLE
chore: removes panics from wallet burn task

### PR DESCRIPTION
**Pull request by SWvheerden**
Description
---
Removes panics from wallet burn task

### CODE REVIEW

**Review by brianp — APPROVED** [review]
LGTM
utAck

File: applications/minotari_console_wallet/src/ui/state/tasks.rs [hunk 1]
```diff
@@ -257,18 +257,19 @@ pub async fn send_burn_transaction_task(
...
-            log::error!("failed to burn minotari: {:?}", err);
-
+    {
+        Ok((burn_tx_id, original_proof)) => (burn_tx_id, original_proof),
+        Err(e) => {
+            error!(target: LOG_TARGET, "failed to burn minotari: {:?}", e);
             result_tx
-                .send(UiTransactionBurnStatus::Error(UiError::from(err).to_string()))
+                .send(UiTransactionBurnStatus::Error(format!("burn error: {}", e)))
                 .unwrap();
```
Comment (hansieodendaal) [inline applications/minotari_console_wallet/src/ui/state/tasks.rs:269]: We could get rid of unwrap as well

File: applications/minotari_console_wallet/src/ui/state/tasks.rs [hunk 2]
```diff
@@ -292,14 +293,38 @@ pub async fn send_burn_transaction_task(
...
+                        let serialized_proof = match serde_json::to_string_pretty(&wrapped_proof) {
+                            Ok(proof) => proof,
+                            Err(e) => {
+                                error!(target: LOG_TARGET, "failed to serialize burn proof: {:?}", e);
+                                result_tx
+                                    .send(UiTransactionBurnStatus::Error(format!(
+                                        "failure to create proof {:?}",
+                                        e
+                                    )))
+                                    .unwrap();
```
Comment (hansieodendaal) [inline applications/minotari_console_wallet/src/ui/state/tasks.rs:305]: ditto unwrap

File: applications/minotari_console_wallet/src/ui/state/tasks.rs [hunk 3]
```diff
@@ -292,14 +293,38 @@ pub async fn send_burn_transaction_task(
...
+                        match std::fs::write(filepath, serialized_proof.as_bytes()) {
+                            Ok(()) => {},
+                            Err(e) => {
+                                error!(target: LOG_TARGET, "failed to write burn proof: {:?}", e);
+                                result_tx
+                                    .send(UiTransactionBurnStatus::Error(format!(
+                                        "failure to write proof {:?}",
+                                        e
+                                    )))
+                                    .unwrap();
```
Comment (hansieodendaal) [inline applications/minotari_console_wallet/src/ui/state/tasks.rs:324]: ditto unwrap

File: applications/minotari_console_wallet/src/ui/state/tasks.rs (hunk 1 above)
Comment (SWvheerden) [reply applications/minotari_console_wallet/src/ui/state/tasks.rs:269]: I think these are fine as a panic. 
The send will only fail if there is not listener on the channel. If there is no listener on the channel, there is a problem. There should be a listener on this channel.

**Pull request merged**